src/main/java/org/tron/core/capsule/StorageRowCapsule.java

```diff
@@ -48,6 +48,12 @@ public StorageRowCapsule(byte[] rowKey, byte[] rowValue) {
     markDirty();
   }
 
+  public StorageRowCapsule(StorageRowCapsule rowCapsule) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this constructor to comply with Java Code Conventions. ([squid:S1213](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1213))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1667&issues=AWbODG2KLpqqSUlcKsS7&open=AWbODG2KLpqqSUlcKsS7)

[ISSUE_KEY]: AWbODG2KLpqqSUlcKsS7